packages/auth/src/platform_cordova/popup_redirect/events.test.ts

```diff
@@ -73,30 +70,30 @@ describe('platform_cordova/popup_redirect/events', () => {
 
   describe('_savePartialEvent', () => {
     it('sets the event', async () => {
+      const spy = sinon.spy(Storage.prototype, 'setItem');
       const event = _generateNewEvent(auth, AuthEventType.REAUTH_VIA_REDIRECT);
       await _savePartialEvent(auth, event);
-      expect(storageStub.setItem).to.have.been.calledWith(
-        'firebase:authEvent:test-api-key:test-app',
-        JSON.stringify(event)
-      );
+      expect(spy.calledWith('firebase:authEvent:test-api-key:test-app',
```

Comment:
Why is this not using sinonChai anymore?
i.e.
```js
expect(spy).to.have.been.calledWith(...)
```

Reply:
Updated to the syntax you mentioned